articles/build-a-jekyll-site/index.md

```diff
@@ -0,0 +1,358 @@
+Up to this stage, your post doesn't have a navbar or a footer. These parts are an essential part of your site and should be on every page.
+
+Create an `_includes` folder in your base directory and create a `navbar.html` file inside. Put the following content inside it.
+
+```html
+<style>
+ul {
+  list-style-type: none;
+  display: flex;
+}
+li{
+ margin: 5px;
+}
+</style>
+
+<ul>
+  <li><a href="/">Home</a></li>
+  <li><a href="/about">About</a></li>
+</ul>
+```
+
+And then include your navbar at the top of your post layout file.
+
+```jinja
+{% include navbar.html %}
+```
+
+Save the files and reload the page to see the changes. The simple navbar in the `navbar.html` file will now be shown in each file that adopts the post layout. You can add the footer the same way.
+
+### Step 8 -- Adding custom styling to your blog
+Up to this stage, you are using the default `minima` theme provided by Jekyll. To style the site from scratch, delete the `theme` property in `_config.yml`.
+
+If you reload your site, it probably looks broken right now. Let's go ahead and rectify that. Create the following files under `_layouts`.
+
+- `default.html`
+- `page.html`
+- `home.html`
+
+Put the following content inside the files.
+
+\# default.html
+```jinja
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>{{page.title}}</title>
+</head>
+<body>
+	{% include navbar.html %}
+    {{ content }}
+</body>
+</html>
+```
+
+\# page.html
+```html
+---
+layout: default
+---
+{{content}}
+```
+
+\# home.html
+```jinja
+---
+layout: default
+---
+
+<ul>
+    {% for post in site.posts %}
+        <li><a href="{{ post.url | relative_url }}">{{post.title}}</a></li>
+    {% endfor %}
+</ul>
+<h3><a href="/about">About</a></h3>
+```
+
+Since Jekyll allows layout nesting, you can use a layout in another layout. If you reload the site at this point, it's not broken but it's just plain HTML. Let's style the home page.
+
+Create a `_sass` folder in your base directory and create a `main.scss` inside it. Also create a `css` folder inside the `assets` folder and create a `styles.scss`. Put the following content inside files.
+
+\# styles.scss
+```scss
+---
+---
+
+@import "main";
+```
+
+\# main.scss
+
+```scss
+body {
+  background-color: rgb(20, 20, 20);
+  color: #fff;
+}
+a{
+    color: rgb(2, 170, 38);
+    text-decoration: none;
+}
+```
+
+The empty front matter in `main.css` tells Jekyll that the file should be processed. Now, import the stylesheet in `default.html`. Put the following line between the `head` tags in `default.html`.
+
+```html
+<link rel="stylesheet" href="/assets/css/styles.css">
+```
+
+Note that the imported stylesheet is a `css` file and not `scss`. The imported file is the processed file located at `_site/assets/css/style.css`. Reload the page to see the stylesheet at work.
+
+> Jekyll uses [Sass](https://sass-lang.com/) stylesheet. You can learn Sass basics [here](https://sass-lang.com/guide).
```

Comment:
stylesheet**s**. You should move this explanation to just before you mention Sass to explain this to users earlier.